README.md

```diff
@@ -48,60 +48,66 @@ feat: added authentication using supabase auth
 
 [learn more about semantic commits](https://www.conventionalcommits.org/en/v1.0.0/)
 
-# How to start Frontend (Next.js + TailwindCSS)
+## How to start Frontend (Next.js + TailwindCSS)
 
-### Steps:
+Make sure you have [git](https://git-scm.com/) and [npm](https://docs.npmjs.com/cli/init) installed in your local machine.
 
-`#1` Make sure you have installed [npm](https://docs.npmjs.com/cli/init) in your local machine where you want to start app
+The repository has a `.vscode` folder that contains `settings.json` and `extensions.json`. The `settings.json` file configures your VS Code editor to use `eslint` and `prettier` on every code save action (`ctrl + s`). The `extension.json` file contains a list of VS Code extensions, VS Code will show these extensions as suggestions in the extensions tab. After installing these extensions, auto linting and formatting should start working.
 
-`#2` Run `npm install` to install all the required packages
+### Steps:
 
-`#3` The repository has a `.vscode` folder that contains `settings.json` and `extensions.json`. The `settings.json` file configures your VS Code editor to use `eslint` and `prettier` on every code save action (`ctrl + s`). The `extension.json` file contains a list of VS Code extensions, VS Code will show these extensions as suggestions in the extensions tab. After installing these extensions, auto linting and formatting should start working.
+1. Clone this repo
+   ```sh
+   git clone https://github.com/ykdojo/defaang.git
+   ```
+2. Go into the project root directory
```

Comment:
```suggestion

2. Go into the project root directory
```

Reply:
Prettier formatted it automatically. I think prettier check will fail if I remove it.

Reply:
I ran `npm run format:fix` on this and it seems to add a newline before and after correctly. Can you run this cmd and see it fixes the formatting for you?